pkg/sql/parser/sql.y

```diff
@@ -2804,6 +2806,18 @@ show_users_stmt:
   }
 | SHOW USERS error // SHOW HELP: SHOW USERS
 
+// %Help: SHOW ROLES - list defined roles
+// %Category: Priv
+// %Text: SHOW ROLES
+// %SeeAlso: CREATE ROLES DROP ROLES WEBDOCS/show-roles.html
```

Comment:
Is it bad form to link to those before they're in? The plan is not to publicize roles until the whole thing is finished.

Reply:
replaced with a TODO. The lint check was failing on 404 anyway.